### DISCUSSION TITLE
Do we need the AMD GPU and ROCm to run ZLUDA?

### DISCUSSION THREAD

**Discussion by aroobam** [vosen/ZLUDA#194] [Q&A] [answered]
I have an Nvidia CUDA binary. I extracted the ZLUDA linux package and tried to run the CUDA binary, but it does not work as expected. This is on a simple Debian server with no GPU and no ROCm installed. (I know, duh!!) I am just trying to see if ZLUDA falls back to CPU or something. So, yeah, the question is, is that a valid expectation or do we need an AMD GPU and ROCm installed to get ZLUDA working? Is there any way to build a ZLUDA version that offloads to CPU (!?). It might all sound stupid at this point, but I have some other things in mind for which these stupid steps are the first ones. Please let me know. Thanks.

**Comment by vosen** [comment, marked as answer]
ZLUDA is closely tied to ROCm and AMD GPUs. Short of rewriting significant parts of code there's no way to target CPUs
